prvni-krucky/cykly/cyklus-for.md

```diff
@@ -60,9 +60,11 @@ mozkové závity.
 Ukažme si například jak se pomocí cyklu spočítá součet všech čísel v seznamu.
 
 ```python
+cisla = [2, 2*2, -1, 50, 0, 42, 9/3]
 soucet = 0
 for cislo in cisla:
     soucet = soucet + cislo
+print(f"Součet: {soucet}" )
```

Comment:
Ty f-stringy jsou jen ve cteni na doma [1], tak je otazka, jestli je bezpecne to takto ukazat, kdyz by to lektor predtim preskocil a opravdu nechal na doma.

[1] https://kodim.cz/kurzy/uvod-do-progr-1/prvni-krucky/vstup-vystup/cteni-na-doma